### DISCUSSION TITLE
Project Rename

### DISCUSSION THREAD

**Discussion by icarusinthewings** [NovaoTech/solaris#20] [Announcements]
Hey Everyone!
I'm here to announce that we're officially considering doing a project rename.
React with a ✨ for Polaris
React with a 🤔 for Phylos.
Thanks for keeping up-to-date with all the changes that are ahead
~ Velocity

**Comment by spazestudios** [comment]
this is a hard one 🤔

**Comment by MikeDev101** [comment]
✨ I fail to see the reason why we should need a rebrand, and Polaris is easier to pronounce imo.

**Reply by MikeDev101** [reply]
edit: I just saw the fact that there are no Polaris domains available. Ouch. I guess I wouldn't mind too much... 🤔

**Comment by CloudGate1234** [comment]
phylos because domain availability and uniqueness
